README.md

```diff
@@ -5,4 +5,5 @@ Common settings and tools for all WebCharts repos
 * [Tests utils](testsUtils/testsUtils.md)
 * [Benchmark infrastructure](benchmark-infrastructure/benchmark-infrastructure.md)
 * The linting rules used with ESLint are the recommended ones from [ESLint Rules](https://eslint.org/docs/rules/)
-* (To be moved here: build scripts)
+* The [building scripts](buildingScripts.md) for webcharts repository
```

Comment:
Thanks! I think you should release a preview npm package out of this branch, install it on webcharts singleKarmaFile branch and try all the tools: build, test, test build, docs... It is not possible to test some features like generating the doc files with npm link. If everything works well then you should merge this PR and release a minor from master.

Reply:
Looks good to me 👍